frontend/src/static/js/v_summary.js

```diff
@@ -89,12 +89,20 @@ window.vSummary = {
       if (this.tmpFilterSinceDate && this.tmpFilterSinceDate >= this.minDate) {
         this.filterSinceDate = this.tmpFilterSinceDate;
         this.getFiltered();
+      } else if (!this.tmpFilterSinceDate) { // If user clears the since date field
+        this.filterSinceDate = this.minDate;
+        this.tmpFilterSinceDate = this.filterSinceDate;
+        this.getFiltered();
       }
     },
     tmpFilterUntilDate() {
       if (this.tmpFilterUntilDate && this.tmpFilterUntilDate <= this.maxDate) {
         this.filterUntilDate = this.tmpFilterUntilDate;
         this.getFiltered();
+      } else if (!this.tmpFilterUntilDate) { // If user clears the until date field
+        this.filterUntilDate = this.maxDate;
+        this.tmpFilterUntilDate = this.filterUntilDate;
+        this.getFiltered();
```

Comment:
can just move the existing `this.getFiltered();` to the end of the function, no need for 2 instance of it since it needs to be executed in both cases.